app/lib/features/pins/pages/pin_page.dart

```diff
@@ -203,4 +216,96 @@ class PinPage extends ConsumerWidget {
       ),
     );
   }
+
+  // attachment list UI
+  Widget _buildAttachmentList(ActerPin pin) {
+    final attachmentTitleTextStyle = Theme.of(context).textTheme.labelLarge;
+    final attachments = ref.watch(pinAttachmentsProvider(pin));
+    final membership = ref.watch(roomMembershipProvider(pin.roomIdStr()));
+    final canPostAttachments = [];
+    if (membership.valueOrNull != null) {
+      final member = membership.requireValue!;
+      if (member.canString('CanPostPin')) {
+        canPostAttachments.add(_buildAddAttachment(pin));
+      }
+    }
```

Comment:
It's still bound to pin yes, I wasn't sure how the ```standalone widget``` would serve in UI terms since the attachment component design could turn out differently for different features. If this is not the case, I can make one :)

Reply:
>  since the attachment component design could turn out differently for different features. If this is not the case, I can make one :)

Already for easier user adoption we should keep it similar across different parts of the UI, so: yes, for now (at least) it will look the same everywhere.

Reply:
Fixed by 2e19583